Tasks/Common/utility-common-v2/tinyGuidUtility.ts

```diff
@@ -0,0 +1,7 @@
+export function tinyGuid() {
```

Comment:
you would need to upgrade the version of common utility, and also each task that utilizes this common utility.

Reply:
done